### ISSUE TITLE
Accessibility evaluation (WCAG 2.1)

### ISSUE THREAD

**Issue by Malvoz** [openlayers/openlayers#11678]
What constitutes a truly accessible web map requires extensive user research, however as can be seen in the [Web map tools WCAG 2.1 evaluation](https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md) (presented by Nic Chan at the 2020 [W3C/OGC Joint Workshop Series on Maps for the Web](https://www.w3.org/2020/maps/), available in [video](https://www.youtube.com/watch?v=V-Gx0e5Gkzg) format and [slides](https://wcag-maps.nicchan.me/)) there are quite a few opportunities to improve accessibility in the UI of OpenLayers maps that would immediately benefit the end-user.

In the evaluation report I have deliberately not included any [techniques](https://www.w3.org/WAI/WCAG21/Techniques/) sufficient to pass a particular <a href="https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md#evaluated-success-criteria">Success Criterion</a> for brevity, however I am hopeful that the community can come together, and improve the state of accessibility. Feel free to ping me about any uncertainties.

Thanks.

/cc @ahocevar (thanks for participating in the workshop!)

**Comment by Malvoz** [comment]
I've submitted issue https://github.com/openlayers/openlayers/issues/12041 to address the failure of [SC 1.1.1 Non-text Content](https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md#111-non-text-content-level-a).

**Comment by Malvoz** [comment]
I've submitted issue https://github.com/openlayers/openlayers/issues/12311 to address the failure of [SC 2.5.5 Target Size](https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md#255-target-size-level-aaa).

**Comment by Malvoz** [comment]
I've submitted issue https://github.com/openlayers/openlayers/issues/12314 to address one failure of SC [1.3.1 Info and Relationships](https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md#131-info-and-relationships-level-a):

> - The web map's semantic _structure_ as a distinct piece of content cannot be programmatically determined.

**Comment by Malvoz** [comment]
I've submitted https://github.com/openlayers/openlayers/issues/12316 to address another failure of SC 1.3.1 [Info and Relationships](https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md#131-info-and-relationships-level-a):

> - The zoom controls' disabled _state_ cannot be programmatically determined.

**Comment by nein09** [comment]
It's possible that OpenLayers will not adopt these recommendations but I'm actually using them to try and make an OpenLayers map that I've developing more accessible, so thank you for this.
